### PR TITLE
Select properly leads to presenter view

### PR DESCRIPTION
**Pull request by thedeagler**

### CODE REVIEW

File: modules/main_Presenter/client/index.jsx [hunk 1]
```diff
@@ -82,10 +80,17 @@ let Presenter = React.createClass({
...
+    }
+
     return (
       <div className="container">
         {
-          this.props.presentation ?
+          this.props.presenter.getIn(['presentation', 'isReady']) ?
           <div className="presenterSlide">
             Current Slide
             <Slides
```
Comment (kimberlygha) [inline modules/main_Presenter/client/index.jsx:96]: closing /> for slide?

File: modules/main_Presenter/client/index.jsx [hunk 2]
```diff
@@ -13,43 +13,39 @@ import Code from 'sub_SharingCode/client/index'
...
-        // react.props.setPresentation(gid);
-        // window.open('/projector/' + gid);
-      }
-    })
   },
 
   componentDidMount() {
     const context = this
-    Presentation.find({gid: this.props.presentation}).observe({
+    Presentations.find({gid: this.props.presentation}).observe({
```
Comment (yinghaochan) [inline modules/main_Presenter/client/index.jsx:27]: I don't think we're using Presentations any more?

Deck stores the slides,
while each time it's run, you create a new show document

File: modules/main_Presenter/client/index.jsx (hunk 1 above)
Comment (thedeagler) [inline modules/main_Presenter/client/index.jsx:96]: Its on line 99.

File: modules/main_Presenter/client/index.jsx (hunk 2 above)
Comment (thedeagler) [inline modules/main_Presenter/client/index.jsx:27]: I'll look at it when I come home. At least it currently works.

**Pull request merged**